crates/autopilot/src/arguments.rs

```diff
@@ -298,6 +302,39 @@ impl std::fmt::Display for Arguments {
     }
 }
 
+#[derive(clap::Parser, Clone)]
+pub struct QuoteDeviationPolicy {
+    /// How much of the order's surplus should be taken as a protocol fee.
+    #[clap(
+        long,
+        env,
+        default_value = "0",
+        value_parser = shared::arguments::parse_percentage_factor
+    )]
+    pub protocol_fee_factor: f64,
+
+    /// Cap protocol fee with a percentage of the order's volume.
```

Comment:
nit: the `--help` text refers to a `percentage` when it's actually a factor that is to be understood as a percentage. Slight difference that bit us in the past.

Reply:
I think we are covered with the parsing function that makes sure it's a factor. We also have a factor in the variable name. I wanted to keep the comments focused on the meaning.